pkg/network/route_cache.go

```diff
@@ -349,3 +359,11 @@ func (n *netlinkRouter) getInterface(srcAddress util.Address, srcIP net.IP, netn
 	routeCacheTelemetry.ifCacheSize.Inc()
 	return iff
 }
+
+func netlinkErrIncWithTag(err syscall.Errno) {
```

Comment:
You can likely change `err` to a `error`, and then try type casting to `syscall.Errno`. And then based on that, tag or not.